src/BoundingBox.h

```diff
@@ -1,8 +1,10 @@
 #ifndef BOUNDING_BOX_H
 #define BOUNDING_BOX_H
 
-#define EIGEN_DONT_ALIGN_STATICALLY
-#include <Eigen/Dense>
+#include "glm/glm.hpp"
+#include "glm/vec4.hpp"

```

Comment:
Honestly, not sure, copy-pasted from another file as part of getting it working.  But it's probably not necessary if you don't think it is, I'll probably do a cleanup pass on stuff like that later today